File: base-forge-security-spring-boot-starter/src/main/java/com/gewuyou/security/exception/SignInIdNotFoundException.java
package com.gewuyou.security.exception;

import org.springframework.security.core.AuthenticationException;

/**
 * 登录 ID 未找到异常
 *
 * @author gewuyou
 * @since 2024-11-05 19:38:51
 */
public class SignInIdNotFoundException extends AuthenticationException {
    public SignInIdNotFoundException(String msg, Throwable cause) {
        super(msg, cause);
    }

    public SignInIdNotFoundException(String msg) {
        super(msg);
    }
}

File: base-forge-security-authentication-spring-boot-starter/src/main/kotlin/com/gewuyou/baseforge/security/authentication/entities/exception/SignInIdNotFoundException.kt
package com.gewuyou.baseforge.security.authentication.entities.exception

import com.gewuyou.baseforge.autoconfigure.i18n.entity.ResponseInformation

/**
 * 登录 ID 未找到异常
 *
 * @author gewuyou
 * @since 2024-12-31 00:04:22
 */
class SignInIdNotFoundException(responseInformation: ResponseInformation) :
    AuthenticationException(responseInformation)
